distributed/diagnostics/__init__.py

```diff
@@ -1,6 +1,6 @@
 from ..utils import ignoring
 from .graph_layout import GraphLayout
-from .plugin import SchedulerPlugin
```

Comment:
Renaming this module may cause some pain for users who use it currently.  

We would have to either provide a period of backwards compatibility, or we just keep the SchedulerPlugin and WorkerPlugin both in plugin.py (which is my preference).